imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,38 @@
+"""User Profile Models."""
+from django.db import models
+from django.conf import settings
+# from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveProfileManager(models.Manager):
+    """Model manager limited to active profiles."""
+
+    def get_queryset(self):
+        """Filter queryset to get default users."""
+        queryset = super(ActiveProfileManager, self).get_queryset()
+        return queryset.filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Class defining imager profile model."""
+
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, related_name='profile')
+    camera_model = models.CharField(max_length=250)
+    photography_genre = models.TextField(default='')
+    contacts = models.ManyToManyField('self', symmetrical=False, default='')
+    address = models.TextField(default='')
+    website = models.URLField(default='')
+    objects = models.Manager()
```

Comment:
Why are you overriding the objects manager that django models already gives you?